### PR TITLE
COMMERCE-12404 | Automation for COMMERCE-11665

### PR DESCRIPTION
**Pull request by 22LucaGiannotta22**
https://liferay.atlassian.net/browse/COMMERCE-12404

### CODE REVIEW

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase [hunk 1]
```diff
@@ -1400,4 +1400,318 @@ definition {
...
+
+				var sku = StringUtil.extractFirst(${rowList}, ":");
+				var value = StringUtil.extractLast(${rowList}, ":");
+
+				AssertElementPresent(
+					key_entryTitle = ${sku},
+					key_index = ${columnIndex},
+					key_value = ${value},
+					locator1 = "CommerceEntry#TABLE_ENTRY_FIELD_VALUE");
+			}
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase:1494]: Try doing 2 assertions in the for cycle with fixed columnIndex

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase [hunk 2]
```diff
@@ -1400,4 +1400,318 @@ definition {
...
+		}
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase:1541]: Same as above

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase [hunk 3]
```diff
@@ -1400,4 +1400,318 @@ definition {
...
+	}
+
+	@description = "COMMERCE-12531. It is possible for the admin to view informations about UOM and quantity for each order item."
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase:1544]:
```suggestion
	@description = "COMMERCE-12531. It is possible for the admin to view information about UOM and quantity for each order item."
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase [hunk 4]
```diff
@@ -1400,4 +1400,318 @@ definition {
...
+	@priority = 5
+	test CanViewUOMAndQuantityOfOrderItemsAsOrderManager {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase:1546]:
```suggestion
	test CanOrderManagerViewOrderItemUOMAndQuantity {
```

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase [hunk 5]
```diff
@@ -1400,4 +1400,318 @@ definition {
...
+
+				var sku = StringUtil.extractFirst(${rowList}, ":");
+				var value = StringUtil.extractLast(${rowList}, ":");
+
+				AssertElementPresent(
+					key_entryTitle = ${sku},
+					key_index = ${columnIndex},
+					key_value = ${value},
+					locator1 = "CommerceEntry#TABLE_ENTRY_FIELD_VALUE");
+			}
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase:1626]: Same as above

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase [hunk 6]
```diff
@@ -1400,4 +1400,318 @@ definition {
...
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase:1657]: Here I think you can do this without the for cycle, just 2 assertions with fixed columnIndex

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase [hunk 7]
```diff
@@ -1400,4 +1400,318 @@ definition {
...
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase:1682]: Same as above

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase [hunk 8]
```diff
@@ -1400,4 +1400,318 @@ definition {
...
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase:1713]: Same as above

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]
@22LucaGiannotta22 please take a look at the comments I left, thanks!

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Review by baharturk — APPROVED** [review]

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

**Comment by 22LucaGiannotta22** [comment]
ci:forward

**Pull request closed without merging**